docker/Dockerfile

```diff
@@ -1,4 +1,34 @@
-FROM ubuntu:xenial
+FROM ubuntu:xenial as builder
+RUN apt-get update -y && apt-get install -y build-essential curl
+
+FROM builder as lshw
+COPY build-lshw.sh /tmp/osie/build.sh
+RUN cd /tmp/osie/ && DESTDIR=/tmp/out ./build.sh
+
+FROM builder as nvme
+COPY build-nvme-cli.sh /tmp/osie/build.sh
+RUN cd /tmp/osie/ && DESTDIR=/tmp/out ./build.sh
+
+FROM builder as mstflint
+COPY build-mstflint.sh /tmp/osie/build.sh
+RUN cd /tmp/osie/ && DESTDIR=/tmp/out ./build.sh
+
+FROM builder as smartmontools
+COPY build-smartmontools.sh /tmp/osie/build.sh
+RUN cd /tmp/osie/ && DESTDIR=/tmp/out ./build.sh
+
+FROM builder as racadm
+RUN apt-get update && apt-get install -y alien wget
+RUN rpm --verbose --import https://linux.dell.com/repo/pgp_pubkeys/0x1285491434D8786F.asc
+WORKDIR /tmp/osie
+RUN wget \
+        https://dl.dell.com/FOLDER05920767M/1/DellEMC-iDRACTools-Web-LX-9.4.0-3732_A00.tar.gz \
+        http://linux.dell.com/repo/community/openmanage/940/bionic/pool/main/s/srvadmin-omilcore/srvadmin-omilcore_9.4.0_amd64.deb
+RUN tar -xvf DellEMC-iDRACTools-Web-LX-9.4.0-3732_A00.tar.gz
+RUN alien iDRACTools/racadm/RHEL8/x86_64/*.rpm
```

Comment:
Use ./*glob* or -- *glob* so names with dashes won't become options.

Reply:
that doesn't apply hear since the `iDrac...` does the same effect as `./`